Taskfile.yml

```diff
@@ -35,7 +35,7 @@ vars:
   LATEST_VERSION_TAG:
     sh: git describe --tags $(git rev-list --tags=v2* --max-count=1)
 
-  VERSION_FLAGS: -ldflags "-X {{.PACKAGE}}/internal/version.BuildVersion={{.VERSION}}"
+  VERSION_FLAGS: '"-X {{.PACKAGE}}/internal/version.BuildVersion={{.VERSION}}"'
```

Comment:
Why remove `-ldflags` - it's one of the flags required to set the version in the executable. Doesn't make sense to me to remove this here, then restate is manually everywhere else.

If you have a need for part of this value elsewhere, introduce a different variable for that purpose rather than redefining this one and making things awkward.